Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.4.1-alpine
+
+ENV RUNTIME_PACKAGES="tini" \
+    BUILD_PACKAGES="build-base sqlite-dev" \
+    GEMSTASH_VERSION="1.0.4"
```

Comment:
Addressed in https://github.com/Intellection/docker-gemstash/pull/1/commits/7378d4a705d28f2884a0a1b0dc3a48962a64d3bc.